Sum Root to Leaf Numbers.py

```diff
@@ -0,0 +1,25 @@
+# time complexity is o(n), where n is the size of the input
+# space complexity is o(1)
```

Comment:
Space complexity is wrong please check

Reply:
Yes, Thanks for correcting, space complexity is o(n), Recursive stack space